src/OpenColorIO/Config.cpp

```diff
@@ -2585,4 +2589,129 @@ OIIO_ADD_TEST(Config, range_serialization)
 
 }
 
+OIIO_ADD_TEST(Config, DefaultDisplay)
+{
+    static const std::string SIMPLE_PROFILE =
+        "ocio_profile_version: 1\n"
+        "\n"
+        "search_path: luts\n"
+        "strictparsing: true\n"
+        "luma: [0.2126, 0.7152, 0.0722]\n"
+        "\n"
+        "roles:\n"
+        "  default: raw\n"
+        "  scene_linear: lnh\n"
+        "\n"
+        "displays:\n"
+        "  sRGB:\n"
+        "    - !<View> {name: Raw, colorspace: raw}\n"
+        "  ArtificialDisplay:\n"
+        "    - !<View> {name: Raw, colorspace: raw}\n"
+        "  AdditionalDisplay:\n"
+        "    - !<View> {name: Raw, colorspace: raw}\n"
+        "\n"
+        "active_displays: [sRGB, ArtificialDisplay, AdditionalDisplay]\n"
+        "active_views: []\n"
+        "\n"
+        "colorspaces:\n"
+        "  - !<ColorSpace>\n"
+        "    name: raw\n"
+        "    family: \"\"\n"
+        "    equalitygroup: \"\"\n"
+        "    bitdepth: unknown\n"
+        "    isdata: false\n"
+        "    allocation: uniform\n"
+        "\n"
+        "  - !<ColorSpace>\n"
+        "    name: lnh\n"
+        "    family: \"\"\n"
+        "    equalitygroup: \"\"\n"
+        "    bitdepth: unknown\n"
+        "    isdata: false\n"
+        "    allocation: uniform\n";
+
+    // First, check without environment variable override.
+    {
+        std::istringstream is(SIMPLE_PROFILE);
+        OCIO::ConstConfigRcPtr config;
+        OIIO_CHECK_NO_THROW(config = OCIO::Config::CreateFromStream(is));
+        OIIO_CHECK_EQUAL(std::string(config->getDefaultDisplay()), "sRGB");
+    }
```

Comment:
In order to target the bug fix, you could add:
OIIO_CHECK_EQUAL(config->getNumDisplays(), 0);